packages/react-sdk/src/components/BoardBar/pdf/utils.ts

```diff
@@ -40,7 +40,68 @@ export function image(element: ImageElement, base64content: string): Content {
   };
 }
 
+/**
+ * This ensures that the svg is meeting the following conditions:
+ *
+ * - The width and height are set (required for the canvas to work)
+ * - The svg is actually an svg
+ *   - If the svg is not an svg we return the original string
+ *   - If the svg is detected to be a png or jpeg we return the original string but set the mimetype accordingly
+ *
+ * @param element The element that is being processed
+ * @param svg The base64 encoded svg string
+ * @returns The base64 encoded svg string after the preprocessing
+ */
+function preprocessSvg(element: ImageElement, svg: string): string {
+  if (element.mimeType !== 'image/svg+xml') {
+    return svg;
+  }
+
+  // Convert the svg base64 to the actual svg
+  const svgString = atob(svg);
+
+  // We lied in the past about the mimetype, so we need to check if the svg is actually an svg and otherwise return the original string
+  if (!svgString.includes('<svg')) {
+    element.mimeType = svgString.includes('PNG') ? 'image/png' : 'image/jpeg';
```

Comment:
Are we sure it MUST be a JPEG, if it is not a PNG? We do also allow GIF uploads (and maybe more in the future, wuch as webp etc.).

Reply:
oops no thats a bug I wrote then :)

Reply:
Function is gone. Resolving